client/ayon_hiero/api/lib.py

```diff
@@ -415,36 +415,41 @@ def get_track_openpype_tag(track):
         return None
     for tag in _tags:
         # return only correct tag defined by global name
-        if OPENPYPE_TAG_NAME in tag.name():
+        if AYON_TAG_NAME in tag.name():
             return tag
 
 
-def get_track_openpype_data(track, container_name=None):
+def get_track_ayon_data(track, container_name=None):
     """
-    Get track's openpype tag data.
+    Get track's AYON tag data.
 
     Attributes:
         trackItem (hiero.core.VideoTrack): hiero object
 
     Returns:
-        dict: data found on pype tag
+        dict: data found on the AYON tag
     """
     return_data = {}
     # get pype data tag from track item
-    tag = get_track_openpype_tag(track)
+    tag = get_track_ayon_tag(track)
 
     if not tag:
         return None
 
     # get tag metadata attribute
     tag_data = deepcopy(dict(tag.metadata()))
+    if tag_data.get("tag.json_metadata"):
+        tag_data = json.loads(tag_data["tag.json_metadata"])
```

Comment:
So `json_metadata` completely replaces the tag data? Do we have duplicated data or why is that?